packages/m2m-gateway/src/services/agreementService.ts

```diff
@@ -39,5 +40,21 @@ export function agreementServiceBuilder(clients: PagoPAInteropBeClients) {
         },
       };
     },
+    getAgreement: async (
+      agreementId: AgreementId,
+      ctx: WithLogger<M2MGatewayAppContext>
+    ): Promise<m2mGatewayApi.Agreement> => {
```

Comment:
```suggestion
    async getAgreement (
      agreementId: AgreementId,
      ctx: WithLogger<M2MGatewayAppContext>
    ): Promise<m2mGatewayApi.Agreement> {
```

Just for consistency, as mentioned in #1782